functional/test_consul_config.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+- hosts: localhost
+  gather_facts: no
+  vars:
+    address: consul.local:8500
+    scheme: https
+    consul_token: myAwesomeConsulManagementToken
+  tasks:
+    - name: start with engine disabled
+      hashivault_secret_disable:
+        name: consul
+         
+    - name: make sure test fails when no mount exists
+      hashivault_consul_secret_engine_config:
+        address: "{{ address }}"
+        scheme: "{{ scheme }}"
+        consul_token: "{{ consul_token }}"
+      register: fail_config
+      failed_when: false
+
+    - assert: { that: "{{ fail_config.changed }} == False" }
+    
+    - name: enable database secret engine
+      hashivault_secret_enable:
```

Comment:
this module is deprecated now, please use `hashivault_secret_engine`

Reply:
fyi - fixing for future ppl too https://github.com/TerryHowe/ansible-modules-hashivault/pull/159/commits/c4be6a82735296f2bd13012c1cc973609003e614